org.eclipse.lsp4j.debug/src/main/java/org/eclipse/lsp4j/debug/services/IDebugProtocolClient.java

```diff
@@ -280,4 +282,24 @@ default void memory(MemoryEventArguments args) {
 	default CompletableFuture<RunInTerminalResponse> runInTerminal(RunInTerminalRequestArguments args) {
 		throw new UnsupportedOperationException();
 	}
+
+	/**
+	 * This request is sent from the debug adapter to the client to start a new debug session
+	 * of the same type as the caller.
+	 * <p>
+	 * This request should only be sent if the corresponding client capability
+	 * <code>supportsStartDebuggingRequest</code> is <code>true</code>.
+	 * </p>
+	 * <p>
+	 * A client implementation of startDebugging should start a new debug session (of the same
+	 * type as the caller) in the same way that the caller’s session was started. If the client
+	 * supports hierarchical debug sessions, the newly created session can be treated as a child
+	 * of the caller session.
+	 * </p>
+	 * Since 1.59
+	 */
+	@JsonRequest
+	default CompletableFuture<StartDebuggingRequestArguments> startDebugging(StartDebuggingResponse args) {
```

Comment:
Looks like response type and request args type are switched in signature.

Reply:
Indeed, thanks!